build/Versions.props

```diff
@@ -7,20 +7,12 @@
   
     <!-- Toolset -->
     <RoslynToolsRepoToolsetVersion>1.0.0-beta-62601-02</RoslynToolsRepoToolsetVersion>
-    <DotNetCliVersion>2.1.300-preview2-008212</DotNetCliVersion>
+    <DotNetCliVersion>2.1.300-preview1-008174</DotNetCliVersion>
 
     <!-- Tests -->
     <MicrosoftNETTestSdkVersion>15.5.0-preview-20170727-01</MicrosoftNETTestSdkVersion>
     <XUnitVersion>2.3.1</XUnitVersion>
     <XUnitRunnerVisualStudioVersion>2.3.1</XUnitRunnerVisualStudioVersion>
     <UsingToolXliff>false</UsingToolXliff>
   </PropertyGroup>
-
-  <PropertyGroup>
-    <RestoreSources>
-      $(RestoreSources);
-      https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
-      https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json;
-    </RestoreSources>
-  </PropertyGroup>
```

Comment:
Why move it to nuget.config?

Reply:
So it works better with VS. Otherwise VS displays the global feeds.

Reply:
The <clear /> keeps the project from depending the global state too.

Reply:
This is not how other repos do it though [1]. If VS doesn't display the right feeds we need to fix VS. 
What global state? 

[1] We do not use nuget.config because it can't be overridden by orchestrated/source build. Although this repo is not part of orchestrated build we ought to be consistent.

Reply:
> To point to feed with a reference built as part of the orchestrated build?

Yes.

I think this might be actually fixed by upgrading to the SDK version of the RepoToolset (1.0.0-beta2-62810-01). Try the latest master of symreader-converter.

Reply:
Maybe the problem I had with symreader-converter was an older version. It builds fine now with nothing in the machine or user global nuget.config.